src/clj/game/cards/agendas.clj

```diff
@@ -17,11 +17,12 @@
     {:msg (msg "gain " (count-ice corp) " [Credits]")
      :interactive (req true)
      :effect (effect (gain-credits (count-ice corp))
-                     (update-all-ice))
+               (update-all-ice))
```

Comment:
OMG WHAT ARE YOU DOING TO THE EFFECT INDENT?

What is the convention we use? I thought the old case was our "standard" case.

Reply:
Whoops, good call. My indent settings in vim get wonky in `letfn`s, so I have to manually verify them. Thanks for catching this one.